portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,21 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  private List<String> quotes;
 
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add("Welcome");
+    quotes.add("Bienvenidos");
+    quotes.add("Bonjour");
+    quotes.add("Aloha");
+    quotes.add("Kon'nichiwa");
+  }
```

Comment:
nit: Add a blank line after this. 1 blank line between methods.

Reply:
done